keys.go

```diff
@@ -156,32 +187,53 @@ func (c *Client) CreateKeyWithAliases(ctx context.Context, name string, expirati
 	return c.CreateImportedKeyWithAliases(ctx, name, expiration, "", "", "", extractable, aliases)
 }
 
+// CreateKeyWithAliasesWithSHA1 same as above function, except it uses
+// RSAES OAEP SHA 1 as encryption algorithm.
+func (c *Client) CreateKeyWithAliasesWithSHA1(ctx context.Context, name string, expiration *time.Time, extractable bool, aliases []string) (*Key, error) {
+	return c.CreateImportedKeyWithAliasesWithSHA1(ctx, name, expiration, "", "", "", extractable, aliases)
+}
+
```

Comment:
This shouldn't be needed... you aren't importing anything here so there is no need for import algorithm type.

Reply:
Done.